dictionary.json

```
@@ -15948,18 +16563,13 @@
   },
   {
     "toaq": "she",
-    "type": "predicate",
-    "english": "▯ is the case if ▯ is the case.",
-    "gloss": "if",
+    "type": "modality",
+    "english": "indicative, necessary",
```

```suggestion
    "english": "indicative, necessary; 'must'",
```

idk, maybe this would cause confusion with duaı